{"status": "broken", "name": "Using `keys()` on a relationship, empty result", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "KeysAcceptance"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486030182, "historyId": "46bedc9f517b24b5d82edd495beb0678", "start": 1578486029967, "steps": [{"status": "passed", "start": 1578486029968, "stop": 1578486030173, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486030174, "stop": 1578486030177, "name": "And having executed", "attachments": [{"source": "9be85ddb-a7c3-4abd-a2b8-d26efbcc5eaf-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486030177, "stop": 1578486030179, "name": "When executing query", "attachments": [{"source": "c5103aff-946f-49d5-9e8f-d4f12c4036a4-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "broken", "attachments": [{"source": "4cb7af65-b068-4a47-8208-67d1af5e8381-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486030181, "start": 1578486030180, "statusDetails": {"message": "ResponseError: Unknown function 'keys'\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486030182, "stop": 1578486030182, "name": "And no side effects"}], "statusDetails": {"message": "ResponseError: Unknown function 'keys'\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "uuid": "11aba1c8-e726-4942-aae5-01e562ba8e08"}